_data/companies/kpn.yml

```
@@ -0,0 +1,6 @@
+name: KPN
+wfh: Mandatory
```

```suggestion
wfh: Required
```

Mandatory -> required for consistency in terminology